test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
```

Comment:
Since an in progress PR will change all the occurences of "matching table" by "grouping rules", this must be changed as well in these acceptance tests.